source/Generic/PlayState/PlayState.cs

```diff
@@ -874,6 +873,24 @@ public override IEnumerable<MainMenuItem> GetMainMenuItems(GetMainMenuItemsArgs
                         var featureRemovedCount = PlayniteUtilities.RemoveFeatureFromGames(PlayniteApi, PlayniteApi.MainView.SelectedGames.Distinct(), "[PlayState] Suspend Playtime only");
                         PlayniteApi.Dialogs.ShowMessage(string.Format(ResourceProvider.GetString("LOCPlayState_PlaytimeSuspendRemovedResultsMessage"), featureRemovedCount), "PlayState");
                     }
+                },
+                new MainMenuItem
+                {
+                    Description = ResourceProvider.GetString("LOCPlayState_MenuItemAddToProcessesSuspendDescription"),
+                    MenuSection = "@PlayState",
+                    Action = a => {
+                        var featureAddedCount = PlayniteUtilities.AddFeatureToGames(PlayniteApi, PlayniteApi.MainView.SelectedGames.Distinct(), "[PlayState] Suspend Processes");
```

Comment:
One last thing, since both features to force a PlayState mode are contradictory, can you remove the other feature for the games while adding one of them? For example, remove `[PlayState] Suspend Playtime only"` after adding the `[PlayState] Suspend Processes` feature and also the opposite. The method should already be there in PlayniteUtilities, it's not necessary to show the removal count in the message.

Reply:
On second thought I'll just add it, won't take more than a minute